.circleci/config.yml

```diff
@@ -239,41 +209,68 @@ commands:
           # Setting it to 2 or higher will suppress the warning messages totally.
           name: Set HDF5_DISABLE_VERSION_CHECK environment variable
           command: setx /m HDF5_DISABLE_VERSION_CHECK 1
-      - run:
-          name: Run unit tests
-          command: |
-            conda activate kedro_builder
-            pytest .\tests --ignore .\tests\extras\datasets\spark --ignore .\tests\extras\datasets\tensorflow --ignore tests\framework\session\test_session_hooks.py --no-cov
+      - when:
+          # Run `test_parallel_runner.py` separately because of `Windows fatal exception: stack overflow`
+          condition:
+            equal: ["3.8", <<parameters.python_version>>]
```

Comment:
This conditional behaviour was there before since `win_unit_tests_38` was a bit different from `win_unit_tests_36` and `win_unit_tests_37`.